node/src/Commands.ts

```diff
@@ -1476,3 +1476,10 @@ export function createObjectEncoding(key: string): redis_request.Command {
 export function createObjectFreq(key: string): redis_request.Command {
     return createCommand(RequestType.ObjectFreq, [key]);
 }
+
+/**
+ * @internal
+ */
+export function createObjectIdletime(key: string): redis_request.Command {
```

Comment:
Should be?
```suggestion
export function createObjectIdleTime(key: string): redis_request.Command {
```

Reply:
in python the command function is called object_idletime, in Java its called ObjectIdletime, and the command itself is just two words: OBJECT IDLETIME instead of OBJECT IDLE TIME. So I think it makes more sense to be ObjectIdletime here? Although the protobuf RequestType name is ObjectIdleTime, so its different in different places

Reply:
I protobuf I named it camelCase to be human friendly and easy-readable.
How to name here - up to you, I don't have preferences.